otlp/logs_test.go

```diff
@@ -530,7 +391,7 @@ func TestLogsRequestWithInvalidContentTypeReturnsError(t *testing.T) {
 }
 
 func TestLogsRequestWithInvalidBodyReturnsError(t *testing.T) {
-	bodyBytes := test.RandomBytes(10)
+	bodyBytes := []byte{0x00, 0x01, 0x02, 0x03, 0x04}
```

Comment:
Funny story: 10 random bytes in one local test run [produced a valid request body](https://en.wikipedia.org/wiki/Infinite_monkey_theorem) for OTLP logs.

Reply:
Yep, I've seen that before too 😬